circe.el

```diff
@@ -1245,11 +1251,13 @@ Do not use this directly, use `circe-reconnect'"
       (circe-display-server-message "Connecting...")))
   (setq circe-server-process
         (irc-connect
+         :server-buffer (current-buffer)
```

Comment:
Duplicate, see a few lines below. It's already part of the record.

Reply:
lol, I don't know why i did not see it :D

Reply:
fixed!